File: python/guidos-gorgeous-lasagna/lasagna.py
EXPECTED_BAKE_TIME = 40
PREPARATION_TIME = 2


def bake_time_remaining(time):
    """
     :param time:
     :return: int
     """
    return EXPECTED_BAKE_TIME - time


def preparation_time_in_minutes(layers):
    """
     :param layers:
     :return: int
      """
    return layers * PREPARATION_TIME


def elapsed_time_in_minutes(layers, time):
    """
    :param layers:
    :param time:
    :return: int
    """
    return time + preparation_time_in_minutes(layers)

File: julia/lasagna/lasagna.jl
const expected_bake_time = 60

const preparation_time(layers) = 2 * layers

const remaining_time(time_in_oven) = expected_bake_time - time_in_oven

const total_working_time(layers, time_in_oven) = preparation_time(layers) + time_in_oven
